inc/exportable.class.php

```diff
@@ -111,4 +112,42 @@ public function importChildrenObjects($item, $linker, $subItems, $input) {
             }
         }
     }
+
+    /**
+     * Count sub items
+     *
+     * @param array $input
+     * @param $subItems
+     * @return integer
```

Comment:
Not a big deal but PHP 7 only accept short terms (int, bool) in type hinting so it can be nice to be consistent with it.
Also recommended in [PSR 12](https://www.php-fig.org/psr/psr-12/#25-keywords-and-types).